plugins/lua/stockpiles.lua

```diff
@@ -508,6 +529,17 @@ function StockpilesOverlay:init()
                                 {label='Training', value=false}},
                         initial_option=false,
                         on_change=self:callback('toggleLogisticsFeature', 'train'),
+                    },  widgets.CycleHotkeyLabel{
+                        view_id='forbid',
+                        frame={t=2, l=0},
+                        auto_width=true,
```

Comment:
this might end up with too small a size. CycleHotkeyLabels generally need an explicit width to be set in the frame

Reply:
I expect to do this with `label_width` unless I'm missing some better method within the `frame` specifications? I was interpreting `frame` as the position but I believe I understand the width issue I need to fix.

Reply:
label_width only applies to the label part of the widget. you need to set the frame width to be sure it's wide enough to handle all option labels.

Reply:
Should be addressed now.